validator/testcases/javascript/instanceactions.py

```diff
@@ -444,6 +457,7 @@ def os_file_readTo_callback(argument, traverser, node, wrapper):
     "addLivemark": livemarkCallback,
     "removeLivemark": livemarkCallback,
     "getLivemark": livemarkCallback,
+    "quote": quote_callback,
```

Comment:
This tells the traverser to call `quote_callback` anytime a `quote` method is called. This means that it is unfortunately not just for String arguments but any object. Since we aren't actually running the code we don't really have a good way of knowing if this is a String or not.